extensions/apicurio-registry-avro/deployment/src/main/java/io/quarkus/apicurio/registry/avro/DevServicesApicurioRegistryProcessor.java

```diff
@@ -118,7 +120,21 @@ public void run() {
     }
 
     private String getRegistryUrlConfig(String baseUrl) {
-        return baseUrl + "/apis/registry/v2";
+        if (isCcompat()) {
```

Comment:
I know that ccompat is what is used by apicurio, but maybe let's say: isUsingConfluentCompatibility() (I've found ccompat to be confusing).

Reply:
Yeah, I remember we used "Confluent" before in the Apicurio, but then rather went with `ccompat`.
I'm OK with your `isUsingConfluentCompatibility`.